### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by binodbhusal**
**The following activities were carried out** 
- Set up a new project with webpack
- Set an array of some simple to do tasks
- CSS  loaded by Webpack Style/CSS Loader

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+
```
Comment (rloterh) [inline webpack.config.js:2]: - [ ] HTML Webpack plugin is not properly configured, and similarly, no `.html` is generated during compilation. Kindly ensure the Webpack config file has an HTML file configured in the Webpack config settings as a template using the HTML Webpack Plugin.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "hint": "^7.1.8",
+    "style-loader": "^3.3.2",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack": "^5.81.0",
+    "webpack-cli": "^5.0.2"
+  }
+}
```
Comment (rloterh) [inline package.json:35]: - [ ] Webpack dev server is not properly configured. Kindly ensure that basic commands for building Webpack in development mode, production mode is configured by adding the necessary script commands.

**Error starting Webpack dev server:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Binod, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 

## Highlights 👍  
🎲 Good job so far creating project structure!
🎲 ReadMe looks good!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- Please note that we intend this app run on a Webpack dev server and there, kindly make the necessary configurations and include appropriate instructions in your ReadMe on how to _setup_ and _run_ your app, under the _Getting Started_ section.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — COMMENTED** [review]
Hi @binodbhusal 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You did not set up workflow actions for Eslint, so there are no checks for Eslint. Kindly include workflow actions for Eslint. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 3]
```diff
@@ -1 +1,136 @@
...
+### Run tests
+To run tests, run the following command:
+- Track HTML linter errors run:
+
+npx hint .
+
+- Track CSS linter errors run:
+
+npx stylelint "**/*.{css,scss}"
+
```
Comment (rloterh) [inline README.md:86]: - [ ] [OPTIONAL]
Kindly include appropriate instructions in your ReadMe on how to run your app, under the _Getting Started_ section.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Binod,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉 💯 🌟 

## Highlights 🎯 
- Project is properly configured!
- App starts as intended!

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Cheers..Happy coding! 💻  🍷  🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**